.github/workflows/dapp.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   setup-local-hedera:
     name: Dapp Tests
     runs-on: ubuntu-latest
-    timeout-minutes: 35
+    timeout-minutes: 65
```

Comment:
Still concerned about this bump. I'd like this to not be necessary, but if it is we should be clear on what's going on